sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -74,10 +81,36 @@ export const PublicStory = ({ story, settings }: PublicStoryProps) => {
           "& :where(a strong):not(:where([class~='not-prose'] *))": {
             color: safeSiteColor,
           },
+
+          px: '$4',
+
+          '& p': {
```

Comment:
why was this added?

Reply:
if it's to remove the padding around the `p` elements you can use the `className="not-prose"` which will remove all the styles

Reply:
ohh I see, will do that instead - thanks!